live-build/misc/ansible-roles/appliance-build.dcenter/tasks/main.yml

```diff
@@ -62,3 +62,8 @@
     - { regexp: '^RPCMOUNTDOPTS=', line: 'RPCMOUNTDOPTS="--num-threads=5 --manage-gids"' }
 
 - command: systemctl disable bind9.service isc-dhcp-server.service isc-dhcp-server6.service
+
+#
+# delphix-platform installs ntp in a disabled state by default.
+#
+- command: systemctl enable ntp.service
```

Comment:
I'd recommend we use `service` instead of `command` here, but I see that we're not doing that a few lines above.. so perhaps being consistent is better..? I'm good as is, but I feel like using `service` (in both places) would probably be "better" (although, practically, it probably doesn't really matter).

https://docs.ansible.com/ansible/latest/collections/ansible/builtin/service_module.html

Reply:
I don't really have much of an opinion on that, but I've launched a test with your proposed change: http://selfservice.jenkins.delphix.com/job/devops-gate/job/master/job/appliance-build-orchestrator-pre-push/5937/. If that passes I'll update the PR.

Reply:
OK, fair enough.